templates/projects/hello-world/files/common/models/message.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "message",
```

Comment:
Also the template description mentions `Message`, while you are using `message` here.